lib/ledger_sync/adaptors/quickbooks_online/adaptor.rb

```diff
@@ -44,10 +44,10 @@ def initialize(
         end
 
         def find(resource:, id:)
-          url = "#{oauth_base_uri}/#{underscore(resource)}/#{id}"
+          url = "#{oauth_base_uri}/#{resourcify(resource)}/#{id}"
```

Comment:
Why this change instead of using `Util::StringHelpers`?

Reply:
its just `.tr('_', '')`. Didn't feel to me that it needs to be as Util. Should I move it as utility helper?

Reply:
This might be QBO specific though. I'm fine with it like this